app/Meeting.php

```diff
@@ -129,18 +129,13 @@ public function isRunning()
 
     /**
      * End meeting
-     * @return boolean
      */
     public function endMeeting()
     {
         $endParams = new EndMeetingParameters($this->id, $this->moderatorPW);
-        $success   = $this->server->bbb()->endMeeting($endParams)->success();
-        if ($success) {
-            $this->end = date('Y-m-d H:i:s');
-            $this->save();
-        }
-
-        return $success;
+        $this->server->bbb()->endMeeting($endParams)->success();
```

Comment:
The endMeeting method throws a RuntimeException. Shouldn't this error be handled?

Reply:
I added @throws \Exception to the phpdocblock, so it can be handled my the calling function. The returned boolean returns the api result